kolibri/core/assets/src/utils/mirrorText.js

```diff
@@ -0,0 +1,295 @@
+import IntlMessageFormatParser from 'intl-messageformat-parser';
+
+// Adapted from https://github.com/ppatotski/app-localizer
+
+const mapping = {
+  0: '0',
+  ' ': ' ',
+  1: '1',
+  2: '2',
+  3: 'Ƹ',
+  4: '4',
+  5: '5',
+  6: '6',
+  7: '7',
+  8: '8',
+  9: '9',
+  ';': '⁏',
+  '?': '␚',
```

Comment:
Yeah, I copied this from a text mirroring website - I imagine there are many possible improvements!